implementations/micrometer-registry-otlp/src/main/java/io/micrometer/registry/otlp/OtlpCumulativeTimer.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2022 VMware, Inc.
+ * Copyright 2023 VMware, Inc.
```

Comment:
Sorry for bothering you with this mundane thing _again_, @jonatan-ivanov, was the year supposed to change, or should it stay the same as it was originally? Or does it not matter at all?